opencga-analysis/src/main/java/org/opencb/opencga/analysis/rga/RgaManager.java

```diff
@@ -67,12 +68,21 @@ public class RgaManager implements AutoCloseable {
 
     private final IndividualRgaConverter individualRgaConverter;
     private final GeneRgaConverter geneConverter;
+
     private final VariantRgaConverter variantConverter;
 
     private final Logger logger;
 
     private static final int KNOCKOUT_INSERT_BATCH_SIZE = 25;
 
+    private static Map<String, OpenCGAResult<?>> cacheMap;
```

Comment:
It's always better to avoid these static large objects. As the RgaManager is a persisted object in `ClinicalWebService` it'd be better to have this as non static

Reply:
Ok

Reply:
Agree, static is not needed here